test/controllers/rentals_controller_test.rb

```diff
@@ -0,0 +1,102 @@
+require "test_helper"
+
+describe RentalsController do
+
+  describe "index" do
+    it "gets index" do
+
+    get rentals_path
+
+    must_respond_with :success
+    expect(response.header['Content-Type']).must_include 'json'
+    end
+  end # describe index end
+
+  describe "check_out" do
```

Comment:
You should also test what happens if you check out more copies than are in the inventory...